src/components/MenuBars/MenuBars.tsx

```diff
@@ -75,7 +80,14 @@ function MenuBars() {
             <MenuToggle disabled direction="left" toggleStartLabel="Start column mode" toggleStopLabel="End column mode" icon={ColumnIcon} onToggle={() => null} />
             <MenuToggle value={state.timer != null} direction="left" toggleStartLabel="Start timer" toggleStopLabel="Stop timer" icon={TimerIcon} onToggle={toggleTimer} />
             <MenuToggle direction="left" toggleStartLabel="Start voting phase" toggleStopLabel="End voting phase" icon={VoteIcon} onToggle={toggleVoting} />
-            <MenuToggle disabled direction="left" toggleStartLabel="Start focused mode" toggleStopLabel="End focused mode" icon={FocusIcon} onToggle={() => null} />
+            <MenuToggle
```

Comment:
I think there's another icon required here or maybe even another background, to make it more explicit to the user that the focus mode is in active state.
Also we need some sort of tooltip what "focus" means or you could probably describe it in the toast? I don't know what would be the best option. We can discuss this together

Reply:
... also the mouse cursor might change to `pointer` on note hover while in focus mode? Or another cursor... maybe `zoom` or `copy`?  What do you think?